### ISSUE TITLE
Support for multiple languages

### ISSUE THREAD

**Issue by timiwahalahti** [msaari/relevanssi-premium-snowball-stemmer#2]
Would be nice if Stemmer supported multiple languages on sites that have Polylang (or WPML) installed.

**Comment by msaari** [comment]
Yes, I agree it would be nice. It may be possible with a filter function on `pre_option_relevanssi_premium_snowball_stemmer_language`: change the stemmer language on the fly to match the site language. Ready-made support for multiple languages would be better, though.

**Comment by msaari** [comment]
This seems to do the job:

```
add_filter( 'pre_option_relevanssi_premium_snowball_stemmer_language', function( $language ) {
    return relevanssi_get_current_language( false );
} );
```

**Comment by msaari** [comment]
Version 1.6 includes `relevanssi_stem_multiple_languages` filter hook, which does the above trick if you make it return `true`.
